src/adapter/sources.ts

```diff
@@ -571,7 +604,9 @@ export class SourceContainer {
       // whether |sourceUrl| looks like a path and belongs to the workspace.
       const sourceUrl = utils.maybeAbsolutePathToFileUrl(this.rootPath, url);
       const baseUrl = sourceMapUrl.startsWith('data:') ? compiled.url() : sourceMapUrl;
-      const resolvedUrl = utils.completeUrlEscapingRoot(baseUrl, sourceUrl);
+      const resolvedUrl = utils.properlyResolveFileUrl(
+        utils.completeUrlEscapingRoot(baseUrl, sourceUrl),
+      );
```

Comment:
I think I may have fixed what you were seeing in master already (on my Windows machine wfh today). See if this fixes it: https://github.com/microsoft/vscode-js-debug/blob/fd559e0e49661d7c558c3a22e9502eb3114a7679/src/adapter/sources.ts#L575-L577

Reply:
It's not exactly what I need 😔. That leaves the resolvedUrl looking like this:

![image](https://user-images.githubusercontent.com/16889261/72657247-eaf33700-3956-11ea-94ee-c26840130b95.png)


But in order to do the url comparisons I need it to look like this:
![image](https://user-images.githubusercontent.com/16889261/72657197-54267a80-3956-11ea-9449-408ba551ba18.png)